website/docs/reference/node-selection/test-selection-examples.md

```diff
@@ -95,16 +103,36 @@ dbt build --select "orders"
 
 <TabItem value="cautious" label="Cautious mode">
 
-It is possible to prevent tests from running if one or more of its parents is unselected (and therefore unbuilt); we call this "cautious" indirect selection.
+For example, only the "orders" model (or node) and its dependent models will be selected for testing or building:
```

Comment:
This isn't accurate.

This would get it back on track:
```suggestion
In this example, only tests that depend _exclusively_ on the "orders" model will be executed:
```